### ISSUE TITLE
1.2 New smoke-tester for autopush

### ISSUE THREAD

**Issue by bbangert** [mozilla-services/push-okr#40]

**Comment by jrconlin** [comment]
https://github.com/mozilla-services/push-service/issues/73

**Comment by jrconlin** [comment]
https://github.com/mozilla-services/push-test (To be replaced by forked ap-loadtest)

**Comment by jrconlin** [comment]
blocked by final OK from QA, but no further coding required at this time. Closing in favor of additional future issues which will be more specific.
